apps/test-viewer/src/components/Viewer.tsx

```diff
@@ -91,10 +91,10 @@ function useViewCreatorOptions() {
       });
     };
 
-    tileTreesLoaded().finally(() => {
-      void IModelApp.tools.run(FitViewTool.toolId, viewPort, true, false);
-      viewPort.view.setStandardRotation(StandardViewId.Iso);
-    });
+    // tileTreesLoaded().finally(() => {
+    //   void IModelApp.tools.run(FitViewTool.toolId, viewPort, true, false);
+    //   viewPort.view.setStandardRotation(StandardViewId.Iso);
+    // });
```

Comment:
I guess this can be removed.

Reply:
@saskliutas Actually I didnt mean to push this, I reverted it